packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Added
+- SF-1170: Add support for `all()` and `allOff()` methods.
```

Comment:
Maybe add `emitter` or `event listener` support... (there are many parts of flux that change, eg store, etc, might be useful to point out specifically what this is for)